etc/databases/world/updates/updates.sql

```diff
@@ -16609,5 +16609,54 @@ begin not atomic
 
         INSERT INTO applied_updates VALUES ('110220232');
     end if;
+
+    -- 12/02/2023 1
+    if (select count(*) from applied_updates where id = '120220231') = 0 then
+        -- remove placeholder objects from Elwynn Forest
+        UPDATE `spawns_gameobjects` SET `ignored` = 1 WHERE `spawn_entry` = 176793;
+        -- remove placeholder objects from Valley of Trials, Durotar
+        UPDATE `spawns_gameobjects` SET `ignored` = 1 WHERE `spawn_entry` = 175784;
+
+        INSERT INTO applied_updates VALUES ('120220231');
+    end if;
+
+    -- 13/02/2023 1
+    if (select count(*) from applied_updates where id = '130220231') = 0 then
+        -- Fix Z position of Lumberjack at Sentinel Hill, Westfall
+        UPDATE `spawns_creatures` SET `position_z` = 34.506 WHERE `spawn_id` = 45524;
+        -- Fix Grayson's Torch display ID
+        UPDATE `item_template` SET `display_id` = 6520 WHERE `entry` = 1172;
+
+        INSERT INTO applied_updates VALUES ('130220231');
+    end if;
+
+    -- 14/02/2023 1
+    if (select count(*) from applied_updates where id = '140220231') = 0 then
+        -- change Father Gavin's level and faction        
+        UPDATE `creature_template` SET `level_min` = 15, `level_max` = 15, `faction` = 35 WHERE `entry` = 1253;
+        -- change Senir Whitebeard's level to 9
+        UPDATE `creature_template` SET `level_min` = 9, `level_max` = 9 WHERE `entry` = 1252;
```

Comment:
See https://github.com/The-Alpha-Project/alpha-core/issues/823#issuecomment-1433614304